.github/workflows/build.yml

```diff
@@ -0,0 +1,11 @@
+name: Build on Ubuntu 22.04
+on: [push]
```

Comment:
This will apply to every branch, which isn't ideal, especially if those are WIP / drafts. In my opinion, master branch pushing is fine and PRs to either master or the 10.6 release branch.